src/theme/theme.css

```diff
@@ -0,0 +1,26 @@
+:root {
+    --color-main: #15a2e7;
+    --color-secondary: #deafe5;
+
+    --font-main: "DejaVu Sans", serif;
+}
+
+button {
+    --color-secondary: #cba64f;
```

Comment:
Are `--color-main` and the other variables representative css variable names or are they the ones controlling ag-grid as defined in the ag-grid docs?

There are two options:
1. project style file defines general variables names with what we allow to customise, like --color-main, and we use another internal style file to map these variables into framework specific styles, such as ag-grid or mui.
2. project style file contains the styling instructions for each framework we use

I favour No. 1.

Reply:
These are general variables. We can also add ag-grid varibles as you suggested. No. 1 option you sounds great.